site-src/geps/gep-1324.md

```diff
@@ -127,6 +127,34 @@ A service **consumer** is someone that utilizes a service by communicating with
 
 Consumer and producer are personas, not concrete workloads or resources. One important aspect of these personas is that they may live in different trust boundaries (typically namespace).
 
+### **Gateway**
+
+We use the term “gateway” in reference to an abstract concept capturing the idea of a component which translates network traffic.
+
+We are using the term in a deliberately abstract way, therefore it is important that readers do not confuse this abstract interpretation of the term - and associated terms - with more concrete interpretations in more specific contexts. For example, we are specifically not referring to the meaning of “gateway” in IP packet routing, nor even the meaning of “translation” in that context.
+
+We can think of a gateway as being a component (aka “network element”) which implements a layer of indirection between a sender of traffic (e.g. a consumer) and its receivers (e.g. a producer). Senders and receivers are all known as “network endpoints”. Gateways consume
```

Comment:
I've been using "translates between somewhere with less context to somewhere with more context", but I think this is probably a better one for this specific use case.